app/views/achievements/_form.html.haml

```diff
@@ -10,17 +10,17 @@
         = f.text_area :description, class: "form-control"
     .form-group
       = f.label :points, t(:achievement_points), class: ["col-sm-2", "control-label"]
-      .col-sm-10
-        = f.text_field :points, class: "form-control"
+      .col-sm-2
+        = f.text_field :points, class: "form-control", type: "number", min: 1, max: 3, size: 5
     .form-group
       = f.label :side, t(:achievement_side), class: ["col-sm-2", "control-label"]
-      .col-sm-10
+      .col-sm-2
```

Comment:
Made this input field -- and the one directly above it -- smaller 'cos they didn't need to stretch across the width of the container.